File: scripts/check_menu.mjs
import { chromium } from 'playwright';

const browser = await chromium.launch({ headless: true });
const page = await browser.newPage({ viewport: { width: 1280, height: 900 } });

await page.goto('http://localhost:3010');
await page.waitForLoadState('networkidle');

// فتح القائمة
const btn = page.locator('button').filter({ hasText: 'القائمة' }).first();
await btn.click();
await page.waitForTimeout(700);

await page.screenshot({ path: 'scripts/menu_shot.png', fullPage: false });
console.log('screenshot saved');
await browser.close();

File: scripts/check_menu.py
from playwright.sync_api import sync_playwright
import time

with sync_playwright() as p:
    browser = p.chromium.launch(headless=True)
    page = browser.new_page(viewport={"width": 1280, "height": 900})
    page.goto("http://localhost:3010")
    page.wait_for_load_state("networkidle")
    
    # Click the menu button
    menu_btn = page.locator("button").filter(has_text="القائمة").first
    menu_btn.click()
    page.wait_for_timeout(600)
    
    page.screenshot(path="scripts/menu_shot.png", full_page=False)
    browser.close()
    print("done")
